maps/away_sites_testing/away_sites_testing.dm

```diff
@@ -1,11 +1,16 @@
 #if !defined(using_map_DATUM)
 
-	#include "..\..\code\content_packages\misc\mundane.dm"
-	#include "..\..\code\content_packages\misc\spacemen.dm"
-	#include "..\..\code\content_packages\corporate\_corporate.dm"
+	#ifndef CONTENT_PACKAGE_MUNDANE
+		#include "..\..\code\content_packages\misc\mundane.dm"
+	#endif
+	#ifndef CONTENT_PACKAGE_SPACEMEN
+		#include "..\..\code\content_packages\misc\spacemen.dm"
+	#endif
+	#ifndef CONTENT_PACKAGE_CORPORATE
```

Comment:
Couldn't these define checks be in the content files themselves?
I.e.
```
#ifndef CONTENT_PACKAGE_CORPORATE
#define CONTENT_PACKAGE_CORPORATE
rest of code in _corporate.dme
#endif

That way they can be included without concern elsewhere

Reply:
Good point.